src/fe/inf_fe_boundary.C

```diff
@@ -78,21 +78,21 @@ void InfFE<Dim,T_radial,T_base>::reinit(const Elem * inf_elem,
       if (s > 0)
         {
           current_fe_type.radial_order = fe_type.radial_order;
-          radial_qrule->init(EDGE2, inf_elem->p_level());
+          radial_qrule->init(EDGE2, inf_elem->p_level(), true);
```

Comment:
I'd prefer it if we had an inline comment for the new hard-coded boolean flags, e.g.
```
radial_qrule->init(EDGE2, inf_elem->p_level(), /*simple_type_only=*/true);
```
but that's a minor comment.